profiles/arch/powerpc/ppc64/64ul/package.use.force

```diff
@@ -0,0 +1,7 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Arthur Zamarin <arthurzam@gentoo.org> (2022-01-13)
+# Bootstrapping is possible using our build snapshot, but force
+# the user to rebuild  
+dev-java/openjdk:17 -system-bootstrap jbootstrap
```

Comment:
this will ALWAYS force user to to internal bootstrap.
we want users to use system version after initial one happened.
need to re-think all that a bit.

Reply:
Actually after rechecking on multiple arches, it looks like the logic for system-bootstrap works:
on amd64 & arm64, when running `equery u openjdk:17`, I don't see system-bootstrap
on ppc64be, when running `equery u openjdk:17`, I see system-bootstrap and I can enable or disable it.

Reply:
yeah I might got confused. should be ok.